packages/sdks/react-sdk/src/components/AuthProvider/AuthProvider.tsx

```diff
@@ -41,6 +41,12 @@ interface IAuthProviderProps {
   refreshCookieName?: string;
   // Function to get external token, for seamless migration from external system
   getExternalToken?: () => Promise<string>;
+  // Custom storage configuration for tokens and user data
+  customStorage?: {
```

Comment:
why don't we take this type from web-js and duplicate it? 

we also borrow this type from web-js
<img width="428" height="119" alt="image" src="https://github.com/user-attachments/assets/98610707-2c05-4cec-b14d-5dc4268f2cd1" />

Reply:
Done